ambry-rest/src/test/java/com/github/ambry/rest/NettyResponseChannelTest.java

```diff
@@ -1591,7 +1626,7 @@ private void setNullHeaders() throws RestServiceException {
    */
   private void setRequestTest() throws RestServiceException {
     ResponseStatus status = ResponseStatus.Accepted;
-    restResponseChannel = new NettyResponseChannel(ctx, new NettyMetrics(new MetricRegistry()), PERFORMANCE_CONFIG);
+    restResponseChannel = new NettyResponseChannel(ctx, new NettyMetrics(new MetricRegistry()), PERFORMANCE_CONFIG, this.nettyConfig);
```

Comment:
Line formatting

Reply:
Done.

Reply:
actually this line is within the 120 char limit. no change needed.